ios/Sources/AppFeature/AppFeature.swift

```diff
@@ -60,16 +64,37 @@ public let appReducer = Reducer<AppState, AppAction, AppEnvironment>.combine(
             .init()
         }
     ),
-    .init { _, action, _ in
+    .init { state, action, _ in
         switch action {
-        case .home:
+        case .home(let homeAction):
+            switch homeAction {
+            case .showSetting:
+                state.isSettingPresented = true
+            default:
+                break
+            }
             return .none
-        case .media:
+        case .media(let mediaAction):
+            switch mediaAction {
+            case .showSetting:
+                state.isSettingPresented = true
+            default:
+                break
+            }
             return .none
-        case .favorites:
+        case .favorites(let favoritesAction):
+            switch favoritesAction {
+            case .showSetting:
+                state.isSettingPresented = true
+            default:
+                break
+            }
             return .none
```

Comment:
How about this?

```swift
case .home(.showSetting),
     .media(.showSetting),
     .favorites(.showSetting):
    state.isSettingPresented = true
    return .none
case .home:
    return .none
case .media:
    return .none
case .favorites:
    return .none
```

Reply:
This one is simpler. I'll change to this.

Reply:
DONE